index.html

```diff
@@ -44,12 +44,9 @@ <h2 class="small">A search for happenings</h2>
 
     </section>
 
-    <script type="text/javascript">
-        $(document).ready(function () {
 
+    <script src="./assets/js/weather.js"></script>
```

Comment:
move this into the header at the top of the file with all the links and other includes